server/src/main/java/org/apache/iotdb/db/query/reader/chunk/DiskChunkLoader.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.db.query.reader.chunk;
+
+import org.apache.iotdb.db.engine.cache.ChunkCache;
+import org.apache.iotdb.tsfile.file.metadata.ChunkMetaData;
+import org.apache.iotdb.tsfile.read.TsFileSequenceReader;
+import org.apache.iotdb.tsfile.read.common.Chunk;
+import org.apache.iotdb.tsfile.read.controller.IChunkLoader;
+
+import java.io.IOException;
+
+public class DiskChunkLoader implements IChunkLoader {
+
+  private final TsFileSequenceReader reader;
+
+  public DiskChunkLoader(TsFileSequenceReader reader) {
+    this.reader = reader;
+  }
+
+  @Override
+  public Chunk getChunk(ChunkMetaData chunkMetaData) throws IOException {
+    return ChunkCache.getInstance().get(chunkMetaData, reader);
+  }
+
+  @Override
+  public void close() throws IOException {
+    reader.close();
+  }
+
+  @Override
+  public void clear() {
+    // no cache need clear
```

Comment:
i think , this method can delete for  interface, neither memChunk nor diskChunk don't need to clear ,and the cache is  LRU